lib/travis/api/v3/routes.rb

```diff
@@ -22,6 +22,10 @@ module Routes
       end
     end
 
+    resource :builds do
```

Comment:
Is there a reason you'd prefer this Josh? The other endpoints on `/user` (eg `/beta_features`) have the `user.id` in the URI, which you aren't including here. That's why I followed the established `/repos` and `/broadcasts` pattern.